cms/tests/toolbar.py

```diff
@@ -1467,7 +1467,7 @@ def test_edit_page(self):
                 page.pk, page.get_title(language)))
         self.assertContains(
             response,
-            '<div class="cms_plugin cms_plugin-cms-page-changelist-%s cms_render_model cms_render_model_block"><h3>Menu</h3></div>' % page.pk)
+            '<div class="cms_plugin cms_plugin-cms-page-changelist-%s cms_render_model cms_render_model_block">\n        <h3>Menu</h3>' % page.pk)
         self.assertContains(
```

Comment:
On this line, is the deletion of the closing `</div>` at the end of the line deliberate?